src/amo/reducers/addonsByAuthors.js

```diff
@@ -172,6 +174,8 @@ const reducer = (
   switch (action.type) {
     case FETCH_ADDONS_BY_AUTHORS: {
       const newState = deepcopy(state);
+      const { addonType: type } = action.payload;
+      const addonType = getAddonTypeFilter(type);
```

Comment:
Why not having `addonTypeFilter` as a variable here?

Reply:
Or maybe just `typeFilter` or `type`...  And the `joinAuthorNamesAndAddonType()` function should be renamed to `joinAuthorNamesAndType()` or `joinAuthorNamesAndTypeFilter()`:

```js
const { addonType } = action.payload;
const type = getAddonTypeFilter(addonType);

joinAuthorNamesAndType(action.payload.authorUsernames, type)
```

Reply:
Actually, I don't think this update is even needed...
I think I can keep most of this in saga/places where we need to filter